test-fixtures/default-args-echo/src/feature/server.env.ts

```diff
@@ -0,0 +1,9 @@
+import sampleFeature, { serverEnv } from './feature';
+
+sampleFeature.setup(serverEnv, () => {
+    return {
+        echoService: {
+            echo: (s = 'dude, it works!') => s,
```

Comment:
Why do you need two echo services, `echo(s = 'No!')` and `echo(s = 'dude, it works!')`

Reply:
@alisey the first one is for testing serialization/deserialization itself but actually it should be without default value, just optional, fixed

The second one is a part of the feature created to test serialization in real remote API calls

Reply:
I still don't understand why we need the first one if the second one is testing a real remote API call.

Reply:
@alisey the idea was to cover that functionality in two test-suits  - as part of `communication` tests to verify only serialization/deserialization process and as a part of `runtime-node` tests to verify how it works in real communication between envs

But you are right - the second test-case is pretty enough, fixed